toolkit/tools/liveinstaller/liveinstaller.go

```diff
@@ -266,7 +266,7 @@ func calamaresInstall(templateConfigFile string, args imagerArguments) (err erro
 	args.configFile = filepath.Join(calamaresDir, "unattended_config.json")
 
 	launchScript := filepath.Join(calamaresDir, "mariner-install.sh")
-	skuDir := filepath.Join(calamaresDir, "mariner-skus")
+	skuDir := filepath.Join(calamaresDir, "azurelinux-skus")
```

Comment:
Good catch.:)

Reply:
Thanks! Lest this come up in the future for any renaming. Our Calamares module implementation relies on this folder being created/named here. So if we update in the tools we will need a corresponding update to our calamares welcome module.